Sources/WebUI/WebView.swift

```diff
@@ -106,6 +106,13 @@ public struct WebView {
         webView.allowsBackForwardNavigationGestures = allowsBackForwardNavigationGestures
         webView.allowsLinkPreview = allowsLinkPreview
         webView.isRefreshable = isRefreshable
+    }
+
+    /// Loads the argument URL received during WebView initialization only once.
+    ///
+    /// This must be used after applyModifiers.
+    @MainActor
+    func loadsURLAtFirst(to webView: EnhancedWKWebView) {
```

Comment:
I feel like the current method name doesn't accurately reflect its behavior.
How about renaming it to something like `loadInitialURL(in:)`?
```suggestion
    func loadInitialURL(in webView: EnhancedWKWebView) {
```

Reply:
Renamed in [a096b22](https://github.com/cybozu/WebUI/pull/1/commits/a096b22d1fae246c56cc6332241cfb2b90b38f79).

Reply:
I believe the argument label `in` is more appropriate than `to`. Thoughts?

Reply:
Modified in [dbb5b5c](https://github.com/cybozu/WebUI/pull/1/commits/dbb5b5c7782c36c83c8b11c3d25f18dba5753445).